pkgs/development/python-modules/aenum/default.nix

```diff
@@ -0,0 +1,19 @@
+{ stdenv, fetchPypi, buildPythonPackage }:
+
+buildPythonPackage rec {
+  pname = "aenum";
+  version = "1.4.7";
+  name = "${pname}-${version}";
+
+  src = fetchPypi {
+    inherit pname version;
```

Comment:
fetchPypi (defined in `pkgs/top-level/python-packages.nix`) takes pname and version. Do you mean, put the expression in `pkgs/top-level/python-packages.nix`?  I was [asked](https://github.com/NixOS/nixpkgs/pull/22689#issuecomment-279448687) earlier to put the new expressions in `python-modules`